kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/FilterDispatchExecutors.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.internal;
+
+import io.netty.channel.EventLoop;
+
+import io.kroxylicious.proxy.filter.FilterDispatchExecutor;
+
+public class FilterDispatchExecutors {
```

Comment:
I think we can reduce the visibility. of this class if it is moved to io.kroxylicious.proxy.internal.filter

Reply:
I got rid of this class and move the factory method onto `NettyFilterDispatchExecutor`, reducing it's constructor to private and making the class public.

I moved the class to ` io.kroxylicious.proxy.filter` to get access to `InternalCompletableFuture`, maybe we need some change so other packages can use the internal Future/Stage implementations without having to know their internals.